### PR TITLE
fix(eventually-postgres): 141 - different types of aggregates with same ID

### PR DESCRIPTION
**Pull request by to266**
closes https://github.com/eventually-rs/eventually-rs/issues/141

### CODE REVIEW

File: eventually-postgres/src/store.rs [hunk 1]
```diff
@@ -86,12 +86,14 @@ const APPEND: &str = "SELECT * FROM append_to_store($1::text, $2::text, $3, $4,
 const CREATE_AGGREGATE_TYPE: &str = "SELECT * FROM create_aggregate_type($1::text)";
 
 const STREAM: &str = "SELECT e.*
-    FROM events e LEFT JOIN aggregates a ON a.id = e.aggregate_id
+    FROM events e LEFT JOIN aggregates a
+    ON a.id = e.aggregate_id AND a.aggregate_type_id = e.aggregate_type
```
Comment (ar3s3ru) [inline eventually-postgres/src/store.rs:90]: Minor: can you indent this?
```suggestion
        ON a.id = e.aggregate_id AND a.aggregate_type_id = e.aggregate_type
```

File: eventually-postgres/src/store.rs [hunk 2]
```diff
@@ -86,12 +86,14 @@ const APPEND: &str = "SELECT * FROM append_to_store($1::text, $2::text, $3, $4,
...
     WHERE a.aggregate_type_id = $1 AND e.aggregate_id = $2 AND e.version >= $3
     ORDER BY version ASC";
 
 const STREAM_ALL: &str = "SELECT e.*
-    FROM events e LEFT JOIN aggregates a ON a.id = e.aggregate_id
+    FROM events e LEFT JOIN aggregates a
+    ON a.id = e.aggregate_id AND a.aggregate_type_id = e.aggregate_type
```
Comment (ar3s3ru) [inline eventually-postgres/src/store.rs:96]: Same for indenting:
```suggestion
         ON a.id = e.aggregate_id AND a.aggregate_type_id = e.aggregate_type 
```

**Review by ar3s3ru — COMMENTED** [review]
Thanks for this PR :heart: 

Left a couple of minor comments, up to you if you wanna change them or not :+1:

**Comment by to266** [comment]
Done :+1:

**Review by ar3s3ru — APPROVED** [review]
LGTM, thanks again :bowing_man:

**Pull request merged**